assets/js/modules/analytics/components/common/SetupUseSnippetSwitch.js

```diff
@@ -79,5 +79,5 @@ export default function SetupUseSnippetSwitch() {
 			</Fragment>
 		);
 
-	return <UseSnippetSwitch description={ description } isSetup />;
```

Comment:
Same here, remove the `isSetup` property from the `UseSnippetSwitch` component: https://github.com/google/site-kit-wp/blob/5bdc7c059312f18a1464794b3f8a69bf0e5319bb/assets/js/modules/analytics/components/common/UseSnippetSwitch.js#L41

See in IB:
> * In UseSnippetSwitch components of **analytics**, analytics-4 and tagmanager modules:
>   * Remove isSetup prop and the class that it adds conditionally.
>   * Remove unused imports.